frontend/ui/translator.lua

```diff
@@ -80,6 +80,7 @@ local SUPPORTED_LANGUAGES = {
     ky = _("Kyrgyz"),
     lo = _("Lao"),
     la = _("Latin"),
+    ["la-lit"] = _("Latin (liturgical)"),
```

Comment:
Why do you need that one here?
It's not an official lang tag, so I guess Google translate doesn't know anything about "la-lit".
Unless this is used for something else in KOReader ?

Reply:
Sorry. I shouldn't have wanted to go so quick…